libs/compute_cuda/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ if(HPX_WITH_CUDA)
     hpx/compute/cuda/traits/access_target.hpp
     hpx/compute/cuda/transfer.hpp
     hpx/compute/cuda/value_proxy.hpp
+    hpx/compute/detail/get_proxy_type.hpp
+    hpx/compute/detail/iterator.hpp
+    hpx/compute/detail/target_distribution_policy.hpp
```

Comment:
I think we can (generally) exclude headers in `/detail` from the generated global module header.

Reply:
Should I remove all the `detail/` files I added ? In some module they are already there, should I remove them too?
I'm not entirely sure what are those headers for in visual studio but when a header is just forward declaring some objects and including the detail file for the main implementation, don't you want to be able to access the detail file in visual studio too ?

Reply:
The `/detail` headers should be listed in the headers list in any case. I think we should exclude them from the generated module header, though.

Reply:
yes sorry for misunderstanding your comment!